bigbluebutton-html5/imports/api/users/server/handlers/userJoined.js

```diff
@@ -4,6 +4,8 @@ import addUser from '../modifiers/addUser';
 
 export default function handleUserJoined({ body }, meetingId) {
   const user = body;
+
+  user.moderator = user.moderator || false;
```

Comment:
Why do we need to know the role now (and were fine before)?

Reply:
Because viewers don't have this field, and I think it's a bad idea to have different structures for each role, for example if I find by users where that haven't this field the find will .